Dockerfile

```diff
@@ -2,11 +2,16 @@ FROM anapsix/alpine-java
 
 MAINTAINER Wurstmeister 
 
-RUN apk add --update unzip wget curl docker jq coreutils
+RUN echo "http://dl-cdn.alpinelinux.org/alpine/v3.7/main" > /etc/apk/repositories && \
```

Comment:
TBH - we plan to switch to the openjdk alpine image (something like https://github.com/wurstmeister/kafka-docker/pull/290) - so this shouldn't be required.

Reply:
@sscaling would you prefer that I will switch to `openjdk alpine image` ?

Reply:
I'm saying that PR #290 supersedes this PR. So if we merge that, this can be closed.

Reply:
PR #290 is merged. Does this fix your issue?

Reply:
@sscaling Not really if it doesn't build older versions.
See I'm using `wurstmeister/kafka:0.10.0.0`, so if you can rebuild the images somehow so it will actually take effect on the images that would be amazing

Reply:
@YonatanKiron - we're trying to position the current master branch so that we can hopefully unify master and release branches. This issue was raised a while ago: https://github.com/wurstmeister/kafka-docker/issues/163

Reply:
Then, yes I think it will fix my issue!